charts/kubewarden-defaults/Chart.yaml

```diff
@@ -3,12 +3,34 @@ name: kubewarden-defaults
 description: A Helm chart for deploying Kubewarden's default PolicyServer instance
 icon: https://www.kubewarden.io/images/icon-kubewarden.svg
 type: application
+kubeVersion: ">= 1.19.0"
 home: https://www.kubewarden.io/
 maintainers:
 - name: José Guilherme Vanz
   email: jguilhermevanz@suse.com
+keywords:
+  - Security
+  - Infrastructure
+  - Monitoring
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.1.7
+version: 1.1.10
```

Comment:
Same here, `1.2.0`

Reply:
Done in amended commit https://github.com/kubewarden/helm-charts/pull/129/commits/c684e4f2e778d0725cad013bc4fcb429417f74e8